script/install.sh

```diff
@@ -371,13 +388,17 @@ Run \`deactivate\` later on to restore your environment."
 }
 
 install_dlang_installer() {
-    local url tmp
-    url=https://dlang.org/install.sh
+    local tmp mirrors
     tmp=$(mkdtemp)
+    mirrors=(
+        "https://dlang.org/install.sh"
+        "https://downloads.dlang.org/other/install.sh"
+        "https://nightlies.dlang.org/install.sh"
```

Comment:
I would add `https://github.com/dlang/installer/raw/stable/script/install.sh` to the list too, but I wasn't sure whether you would be okay with it.

Reply:
Let's not automatically deploy this, it's not worth the security implications.
Even without sudo (which often is cached), `rm -rf ~` can do a lot of damage.